src/bpmncwpverify/bpmn.py

```diff
@@ -20,6 +21,19 @@ def __init__(self, element: Element) -> None:
         self.element = element
         self.id = element.attrib["id"]
         self.name = element.attrib.get("name")
+        self.cleanup()
+
+    def cleanup(self) -> None:
```

Comment:
Is the `name` attribute what is used in the Promela generation? Here we are removing whitespace etc from the name. But why? Is that so it can be emitted in Promela in a sanitized from that matches propels syntactic requirements?

Reply:
Yes, this is taken from the original code to make the bpmn element names syntactically correct for promela.